jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/IO.java

```diff
@@ -224,36 +224,110 @@ public static void copyDir(File from, File to) throws IOException
     }
 
     /**
-     * Copy the contents of a directory from one directory to another.
+     * Copy the contents of a source directory to destination directory.
      *
      * @param srcDir the source directory
      * @param destDir the destination directory
      * @throws IOException if unable to copy the file
      */
+    public static void copyDir(Path srcDir, Path destDir) throws IOException
```

Comment:
This new helper creates confusion with its sibling that takes a `CopyOption...`. From the user of this API it's a bit confusing to see two equivalent methods. From the maintainer perspective, they're 99% identical but have a subtly different implementation.

Isn't it possible to reconcile them into a single method?

Reply:
Seeing as this impl doesn't use the CopyOptions for anything, but the other one does, I don't see the problem.

Reply:
Can't you merge both methods by adding `if (copyOptions.length == 0)` or equivalent?

Reply:
I'll document, it'll make more sense with javadoc.

Reply:
`copyDir(Path, Path, Copyoption...)` still ticks me. It is only used by a test that could use `copyDir(Path, Path)` instead.

I think we could deprecate it and make the test use `copyDir(Path, Path)`.